content/en/docs/project-administration/project-network-isolation.md

```diff
@@ -0,0 +1,180 @@
+6. Click **OK**
+
+{{< notice note >}}
+If the network still doesn't work after you set the network policy, then you need to check if peer has set the corresponding egress rule.
+{{</ notice >}}
+
+#### Allow egress traffic to workloads selected by service
+
+1. Select **Cluster Internal Allowlist**
+2. Click **Add Allowlist**
+3. Select type **Service**
+4. Select the project `testProject`
+5. Select the service `testService` that you want to allow
+6. Select direction **Egress**
+7. Click **OK**
+
+{{< notice note >}}
+When creating a service you must make sure that the selectors of the service are not empty.
+{{</ notice >}}
+
+### Cluster External
+
+Outside the cluster we distinguish between the peers by CIDR.
+
+Suppose a `tomcat deployment` workload has been created in this project  and is exposed via `NodePort` service `testService` on NodePort `80` with `TCP`.
+A client with ip address `192.168.1.1` will access this service, and the service will have access to the address `http://10.1.0.1:80`.
+
+#### Allow ingress traffic from client outside the cluster
+
+1. Select **Cluster External IP Address**
+2. Select direction **Ingress**
+3. Fill CIDR `192.168.1.1/32`
+4. Select protocol `TCP` and fill port `80`
+5. Click **OK**
+
+{{< notice note >}}
+It is best to set `spec.externalTrafficPolicy` in the service configuration to `local`, so that the source address of the packet will not change, i.e. the source address of the packet is the source address of the client.
+{{</ notice >}}
+
+#### Allow egress traffic to service outside the cluster
+
+1. Select **Cluster External IP Address**
+2. Select direction **Egress**
+3. Fill CIDR `10.1.0.1/32`
+4. Select protocol `TCP` and fill port `80`
+5. Click **OK**
+
+{{< notice note >}}
+In step 4, when you select **SCTP**, you must make sure the SCTP is [enabled](https://kubernetes.io/docs/concepts/services-networking/network-policies/#sctp-support). 
+{{</ notice >}}
+
+### Best practice
+
+To ensure that all pods in the project are secure, a best practice is to enable network isolation. 
+
+When network isolation is on, the project cannot be accessed by other projects.  If your workloads need to be accessed by others, you can follow these steps:
+1. Set gateway via **Project Settings/Advanced Settings**
+2. Expose workloads that need to be accessed to a gateway via a service
+3. Allow ingress traffic from the namespace where you gateway locate
+
+If egress traffic controls, you should have a clear plan of what projects, services, and IP addresses can be accessed, and then add them one by one.
+If you're not sure what you want, then you'd better keep your network policy unchanged.
+
+## FAQ
+
+Q: Why can't custom monitoring get data when I enable network isolation?
+
+A: When you enable custom monitoring, kubesphere-monitoring-system will access the metrics of the pod, you need to allow 
+kubesphere-monitoring-system ingress, otherwise the custom monitoring will can not get the metric.
```

Comment:
->
A: After you enabled custom monitoring, KubeSphere monitoring system will access the metrics of the pod. You need to allow ingress traffic for KubeSphere monitoring system. Otherwise, it cannot access pod metrics.